src/components/BookItem.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/Books/books';
+import { deleteBook } from '../redux/Books/books';
 
 const BookItem = ({ book }) => {
   const {
     booktitle, id, author, category,
```

Comment:
- [ ] Kindly change this from booktitle to title to make it work and display the title of the book.